support/insights-operator/disabling-insights-operator.adoc

```diff
@@ -0,0 +1,23 @@
+[id="disabling-insights-operator"]
+= Disabling the Insights Operator
+include::modules/common-attributes.adoc[]
+:context: insights-operator-disabling
+
+toc::[]
+
+:FeatureName: The Insights Operator
+include::modules/technology-preview.adoc[leveloffset=+0]
+
+By default, the Insights Operator is enabled for {product-title}. To disable data collection by the Insights Operator, you must:
+
+. xref:../../support/insights-operator/disabling-insights-operator.html#insights-operator-new-pull-secret_insights-operator-disabling[Modify the global cluster pull secret] to disable Telemetry and the Insights Operator.
+. xref:../../support/insights-operator/disabling-insights-operator.html#images-update-global-pull-secret_insights-operator-disabling[Update the cluster] to use this modified pull secret.
+
+[WARNING]
+====
+It is not recommended to disable the Insights Operator, which also disables Telemetry. Be sure to review the xref:../../telemetry/opting-out-of-telemetry.html#telemetry-consequences-of-disabling-telemetry_telemetry[consequences of disabling Telemetry] before doing so.
```

Comment:
There are some cases where you you need to disable it, though. I'd make this a prereq instead of a warning. (Also, use .adoc instead of .html in the xref.)